### DISCUSSION TITLE
Create Room: Presets and Defaults

### DISCUSSION THREAD

**Discussion by daniellekirkwood** [element-hq/element-meta#643] [Product Feedback]
There was a discussion [here](https://matrix.to/#/!ZCWYLfAKXolsPWObuZ:matrix.org/$cJiVLreIrNNqExN2Zhb7hwh7uoehE6jpsk7M6YMmrFI?via=matrix.org&via=element.io&via=vector.modular.im) that holds some gems! 

Discord sets out presets when creating rooms and hides a lot of the technicalities when users are setting up a new room... We could apply a similar simplicity so that creating a room just works [TM].

**Comment by thibaultamartin** [comment]
I'm adding the screenshot posted in the conversation just in case not everybody has access to it.

<img width="473" alt="Screenshot 2022-09-16 at 15 15 29" src="https://user-images.githubusercontent.com/1688309/191200891-b3252102-f2df-4946-bef8-d16134e0023e.png">

Roughly, translated, this is a room creation screen. Title means "Create a room".
Below are two sections: "room type" and "room name". The room names are in English. Let me know if you need a translation for each room type.

This room creation screen appeared in a Discord Server (eq of Slack Workspace or Matrix Space) where I'm the admin. It makes the assumption that the room is going to be created in a Discord Server, since Discord doesn't have a concept of "standalone room".

One thing that is not shown in the screenshot but that is implicit:
* Every admin of the Discord server is an admin of the room (not the case in Matrix, and I think it breaks expectations)
* The room cannot "leave" the Server, its ownership cannot be transferred

In Discord every room belongs to one and only Discord Server. DMs and small group discussions belong to a sort of special Server: the "Direct Messages" Server.
